### ISSUE TITLE
Labels with parent info for directories with same name

### ISSUE THREAD

**Issue by mkvlrn** [UrosVuj/Explorer-Bookmark#4]
TL;DR: would it be possible to add labels in front of bookmarked directories to represent their parent-folder in a workspace?

Hey!

I'm trying to move away from [this similar extension](https://marketplace.visualstudio.com/items?itemName=saekiraku.pin-up) that seems abandoned and has stopped working properly, it gets stuck while "searching" for the pinned folders or something, it's a hassle everytime.

Your extension is missing just a simple thing that would be great for workspaces. The picture will detail what I mean:
![Screenshot from 2022-02-01 11-43-33](https://user-images.githubusercontent.com/6718750/151989771-78f9b197-f3f7-4745-bb40-6a23419e7b6f.png)

Thanks!

**Comment by UrosVuj** [comment]
Hey Mike!

Thanks for the feedback. Your idea can be really helpful in certain scenarios, so I'll look into it and I'll ping you when it's done  😃  Until then, one thing you can do is hover a folder and it'll show you its full path so you can differentiate them.
